src/QirRuntime/test/Standalone/InputTypes/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# compile test ll files into a library
+
+set(INPUT_TYPES_FILES
+#  qir-test-arrays
+#  qir-test-functors
+#  qir-test-noqsharp
+#  qir-test-qsharp
+#  qir-test-partials
+#  qir-test-qubits-results
+  qir-input-types
+)
+
+foreach(file ${INPUT_TYPES_FILES})
+  compile_from_qir(${file} "") # don't create a target per file
+  list(APPEND QIR_INPUT_TYPES_LIBS ${QIR_UTILITY_LIB})
+endforeach()
+
+add_custom_target(qir_input_types_lib DEPENDS ${QIR_INPUT_TYPES_LIBS})
+
+#==============================================================================
+# The executable target for QIR tests triggers the custom actions to compile ll files
+#
+add_executable(qir-input-types
+  qir-driver.cpp)
+
+target_link_libraries(qir-input-types PUBLIC
+  ${QIR_INPUT_TYPES_LIBS}
+  ${QIR_BRIDGE_UTILITY_LIB}
+  ${QIR_BRIDGE_QIS_UTILITY_LIB}
+  qir-rt-support
+  qir-qis-support
+  simulators
+)
+
+target_include_directories(qir-input-types PUBLIC
+  "${test_includes}"
```

Comment:
if this sample is meant to be "user code", it probably shouldn't depend on test-specific headers (another reason to have this new code outside of `test` so it's hard to pick up unwanted dependencies)

Reply:
I'll remove this.

Reply:
Removed test dependency.